ush/forecast_postdet.sh

```diff
@@ -326,14 +326,14 @@ FV3_out() {
   restart_dates=()
 
   case ${RUN} in
-    gdas|enkfgdas|enkfgfs) # Copy restarts in the assimilation window for RUN=gdas|enkfgdas|enkfgfs
+    gdas|enkfgdas|enkfgfs|enkfgcafs|gcdas) # Copy restarts in the assimilation window for RUN=gdas|enkfgdas|enkfgfs
```

Comment:
What is `gcdas`?

Reply:
the cycling component. Think gcdas == gdas where gcafs == gfs